src/Security/DefenderForStorage.Autorest/examples/Get-AzSecurityDefenderForStorage.md

```diff
@@ -0,0 +1,21 @@
+### Example 1: Get Defender for Storage V2 settings on a storage account
```

Comment:
There are some issues in those two examples files provided by you, since we could not build the module with those example files. Please fix those example issues.